app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -133,26 +145,43 @@
           @render="onFormRender"
           :language="multiLanguage"
         />
-        <p v-if="version" class="text-right">
+        <p
+          v-if="version"
+          :class="isRTL ? 'text-left' : 'text-right'"
+          class="mt-9"
```

Comment:
Just curious - did you mean to add this `mt-9` margin?

Reply:
Correct